openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -423,9 +430,82 @@ class TaxonomyTagsView(ListAPIView):
         * 400 - Invalid query parameter
         * 403 - Permission denied
         * 404 - Taxonomy not found
+
+    **Create Query Parameters**
+        * pk (required) - The pk of the taxonomy to create a Tag for
```

Comment:
```suggestion
        * id (required) - The ID of the taxonomy to create a Tag for
```

(very minor nit:) Using `pk` to refer to the primary key is kind of an internal django thing that REST API integration developers may not be familiar with. I would simply say `ID` in the documentation strings.

Reply:
Makes sense, updated.